policy/modules/system/systemd.te

```diff
@@ -610,6 +621,7 @@ allow systemd_logind_t systemd_sessions_runtime_t:dir manage_dir_perms;
 allow systemd_logind_t systemd_sessions_runtime_t:file manage_file_perms;
 allow systemd_logind_t systemd_sessions_runtime_t:fifo_file manage_fifo_file_perms;
 
+kernel_dontaudit_getattr_proc(systemd_logind_t)
```

Comment:
This might make sense to be part of systemd_log_parse_environment. Did you check if systemd was throwing errors over this? I've tried several `dontaudits` for systemd and they rarely seem to work

Reply:
In my testing I haven't seen any errors from this, but I don't see a problem with simply allowing the access either.